doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -82,22 +85,72 @@ If you want to add SMP Service to advertising data, refer to the :ref:`zephyr:sm
 
 .. fota_upgrades_over_ble_additional_information_end
 
+.. fota_upgrades_over_ble_mcuboot_direct_xip_information_start
+
+Build configuration additions for MCUboot in the Direct-XIP mode
+----------------------------------------------------------------
+
+FOTA updates are also supported when MCUboot is in the Direct-XIP mode.
+In this mode, the MCUboot bootloader boots an image directly from a given slot, so the swap operation is not needed.
+It can be used either with or without the revert mechanism support.
+For more information about the Direct-XIP mode and the revert mechanism support, go to the :doc:`mcuboot:design` page.
```

Comment:
```suggestion
For more information about the Direct-XIP mode and the revert mechanism support, go to the Equal slots (direct-xip) section on the:doc:`mcuboot:design` page.
```

Reply:
Yes, let's include the full reference just as per original suggestion